azure-pipelines.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         displayName: yarn
 
       - script: |
-          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/react-northstar --pr)
+          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/docs --pr)
```

Comment:
these builds should have been scoped to the relevant tested packages cc @ralucapelin